rust/src/pgsql/pgsql.rs

```diff
@@ -558,8 +563,20 @@ pub unsafe extern "C" fn rs_pgsql_probing_parser_ts(
     if input_len >= 1 && !input.is_null() {
 
         let slice: &[u8] = build_slice!(input, input_len as usize);
-        if probe_ts(slice) {
```

Comment:
Is probe_ts still used ?

Reply:
Yes, it is used by `pgsql::parse_request` to decide whether there's a gap or not. (cf https://github.com/OISF/suricata/blob/master/rust/src/pgsql/pgsql.rs#L316)